### DISCUSSION TITLE
Mermaid not rendering correctly in dark mode

### DISCUSSION THREAD

**Discussion by Alex-apostolo** [pradyunsg/furo#634] [Potential Issue]
I installed Mermaid and trying to use it with Furo, in dark mode I would expect the arrows with their descriptions to be white but that is not the case and they are barely visible. 
![Screenshot 2023-03-16 at 17 33 03](https://user-images.githubusercontent.com/44866858/225704707-cade73bd-6037-4a16-8b5c-a51986c0dbde.png)

One way to solve this issue is by including this css class in the `custom_css` for Sphinx.
```css
.mermaid {
  background: white;
  border-radius: 3px;
  margin: 16px 2px;
}
```
Which produces this result
![Screenshot 2023-03-16 at 17 38 56](https://user-images.githubusercontent.com/44866858/225706072-6832686e-9cc4-4daf-a9ed-7026c12adf0e.png)

But optimally it would look like this

![Screenshot 2023-03-16 at 17 42 37](https://user-images.githubusercontent.com/44866858/225706925-b962a4f3-89ce-46b4-a5e2-c4119be5dd67.png)

I have found similar discussions such as [this](https://gitlab.com/gitlab-org/gitlab/-/issues/225341) but not one for Furo. Any help would be much appreciated

**Comment by kdeldycke** [comment]
Some proposal to fix this issue:
- https://github.com/mgaitan/sphinxcontrib-mermaid/issues/205
- https://github.com/mgaitan/sphinxcontrib-mermaid/issues/78
